backup/main/java/fireraya/main/Fireraya.java

```diff
@@ -0,0 +1,229 @@
+
+                if (keyword.equals("deadline")) {
+                    int index = -1;
+                    int i = 0;
+                    String breakpoint = "/by";
+                    for (String element : all) {
+                        if (element.equals(breakpoint)) {
+                            index = i;
+                            break;
+                        }
+                        i++;
+                    }
+                    if (index == -1) {
+                        throw new FirerayaException("No /by detected in deadline");
+                    }
+                    String deadline = String.join(" ", Arrays.copyOfRange(all, index + 1, arrLen));
+                    String task = String.join(" ", Arrays.copyOfRange(all, 1, index));
+                    addDeadline(task, deadline);
+                    continue;
+                }
+
+
+                if (keyword.equals("event")) {
+                    int indexf = -1;
+                    int indext = -1;
+                    int f = 0;
+                    int t = 0;
+                    String fromPoint = "/from";
+                    String toPoint = "/to";
+
+                    for (String element : all) {
+                        if (element.equals(fromPoint)) {
+                            indexf = f;
+                            break;
+                        }
+                        f++;
+                    }
+                    if (indexf == -1) {
+                        throw new FirerayaException("No /from detected in deadline");
+                    }
+                    for (String element : all) {
+                        if (element.equals(toPoint)) {
+                            indext = t;
+                            break;
+                        }
+                        t++;
+                    }
+                    if (indext == -1) {
+                        throw new FirerayaException("No /to detected in deadline");
+                    }
+
+                    String task = String.join(" ", Arrays.copyOfRange(all, 1, indexf));
+                    String from = String.join(" ", Arrays.copyOfRange(all, indexf + 1, indext));
+                    String to = String.join(" ", Arrays.copyOfRange(all, indext + 1, arrLen));
+                    addEvent(task, from, to);
+                    continue;
+                }
+
+                if (keyword.equals("delete")) {
+                    if (all.length > 2) {
+                        throw new InvalidNumOfArgsException();
+                    }
+                    int curr = Integer.parseInt(all[1]) - 1;
+                    if (tasks.size() <= curr || curr < 0) {
+                        throw new FirerayaException("That task does not exist!");
+                    }
+                    System.out.println("Noted. I've removed this task:");
+                    listTask(curr);
+                    tasks.remove(curr);
+                    taskCount();
+                    continue;
```

Comment:
Try to add space between lines doing a specific task for better readability!
```suggestion
                    if (all.length > 2) {
                        throw new InvalidNumOfArgsException();
                    }
                    
                    int curr = Integer.parseInt(all[1]) - 1;
                    
                    if (tasks.size() <= curr || curr < 0) {
                        throw new FirerayaException("That task does not exist!");
                    }
                    
                    System.out.println("Noted. I've removed this task:");
                    listTask(curr);
                    tasks.remove(curr);
                    taskCount();
                    
                    continue;
```